std/file.d

```diff
@@ -3267,7 +3267,7 @@ void copy(RF, RT)(auto ref RF from, auto ref RT to, PreserveAttributes preserve
 unittest // issue 15319
 {
     import std.file : dirEntries;
-    auto fs = dirEntries(getcwd, SpanMode.depth);
+    auto fs = dirEntries(tempDir(), SpanMode.depth);
```

Comment:
`getcwd` returns `/` when called inside an Android apk, which means `dirEntries` traverses the entire file system from here and errors out on any directory for which it doesn't have permissions.  Since this test doesn't really care what directory is used, use the safer `tempDir()` instead.